app/assets/javascripts/discourse/app/templates/preferences/account.hbs

```diff
@@ -46,12 +46,49 @@
   <div class="control-group pref-email">
     <label class="control-label">{{i18n "user.email.title"}}</label>
     {{#if model.email}}
-      <div class="controls">
-        <span class="static">{{model.email}}</span>
-        {{#if model.can_edit_email}}
-          {{#link-to "preferences.email" class="btn btn-default btn-small btn-icon pad-left no-text"}}{{d-icon "pencil-alt"}}{{/link-to}}
-        {{/if}}
-      </div>
+      {{#if siteSettings.enable_secondary_emails}}
```

Comment:
What happens if the site setting is disabled?

Reply:
It shows the old user interface.

Reply:
Nevermind I missed the `{{else}}` 🙈